packaging/hudi-kafka-connect-bundle/pom.xml

```diff
@@ -107,6 +107,7 @@
                                     <include>com.lmax:disruptor</include>
                                     <include>com.github.davidmoten:guava-mini</include>
                                     <include>com.github.davidmoten:hilbert-curve</include>
+                                    <include>com.github.ben-manes.caffeine:caffeine</include>
```

Comment:
separate PR?

Reply:
+1, we can use a weak reference hash map now and we should add a close hook for the instances.